### DISCUSSION TITLE
Disable automatic abbreviation substitution in transcription.  eg "CM" to "centimeters"

### DISCUSSION THREAD

**Discussion by meu-ai** [deepgram/community#368] [General help] [answered]
My pilot project transcribes audio the often includes "CM" (pron "cee emm") yet the transcription comes out as "centimeters". Can I stop this substitution? 

source audio - "I went to the CM launch"
current incorrect transcription - "I went to the centimeters launch" 

Alternatively are replace or keywords the more appropriate work arounds?
Thanks

**Comment by jpvajda** [comment, marked as answer]
@Alex-MEU 👋  You should be able to use Deepgram's [Keywords feature](
https://developers.deepgram.com/docs/keywords) to allow Deepgram to better handle this specific term.

Our Docs on this feature of some examples of how it works. Let us know if this help you.

[Find & Replace ](https://developers.deepgram.com/docs/find-and-replace)could also help here but it might not be as successful as Keyword boosting.

**Reply by meu-ai** [reply]
thanks @jpvajda
